src/qt/pivx/proposalcard.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2021 The PIVX developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or https://www.opensource.org/licenses/mit-license.php.
+
+#include "qt/pivx/proposalcard.h"
+#include "qt/pivx/forms/ui_proposalcard.h"
+
+#include "qt/pivx/qtutils.h"
+
+ProposalCard::ProposalCard(QWidget *parent) :
+    QWidget(parent),
+    ui(new Ui::ProposalCard)
+{
+    ui->setupUi(this);
+    setStyleSheet(parent->styleSheet());
+    setCssProperty(ui->btnVote, "btn-primary");
+    setCssProperty(ui->card, "card-governance");
+    setCssProperty(ui->labelPropName, "card-title");
+    setCssProperty(ui->labelPropAmount, "card-amount");
+    setCssProperty(ui->labelPropMonths, "card-time");
+    setCssProperty(ui->labelStatus, "card-status-passing");
+    setCssProperty(ui->btnVote, "card-btn-vote");
+    setCssProperty(ui->btnLink, "btn-link");
```

Comment:
That would not be good, UX and design wise. Some points:

1) the URL link is an actionable, it does not provide information to the user until he/she does something with it. it can, and it should, be presented as an standalone icon with a clear tooltip text or, in his absence, it could be presented in a separate popup menu if the icon isn't understandable enough (which a good tooltip text could cover in this case).
The icon action actually was a "open the default browser" but.. i didn't add it because we don't have the functionality on the sources but could easily add it. 

2) The URL can be up to 64 chars long. It is not correct to just fit it adding a much smaller text, not many users will be able to read it properly nor it will be presented properly in the current layout distribution.

3) The idea of adding a copy icon works for information, detailed, dialogs, not for the items of a list/grid (dialogs that are opened from the list/grid items, expanding the basic information that they are presenting). Cannot have every action and text on a single grid/list item view.

Saying that, feedback taken 👍 , will work on this and add a good tooltip text for the icon, then will work on another option seeing how it looks with the two actions sub-menu popup (changing the icon for the menu icon), having both: "open URL in default browser" and "copy URL" actionables with clear texts.

Reply:
Agree with zebra on the icon image being somewhat confusing given that the action taken is to copy the URL to the clipboard. All other "copy to clipboard" actions in the UI use the icon that zebra suggested, so this one is a visual outlier case.

Could maybe add a tooltip to the icon that explains what the button does, which can also double as displaying the proposal URL (ex: "Copy Proposal URL %1 to the clipboard") in `setProposal()`